src/test/java/frc/robot/subsystems/ColorSubsystemTest.java

```diff
@@ -0,0 +1,92 @@
+package frc.robot.subsystems;
+
+import org.junit.Test;
+
+import frc.robot.datasources.ColorSource;
+import com.revrobotics.ColorSensorV3.RawColor;
+import frc.robot.utils.ColorOptions;
+
+import org.junit.Before;
+import static org.junit.Assert.*;
+
+public class ColorSubsystemTest {
+
+  class MockColorSource implements ColorSource {
+
+    private RawColor rawColor;
+
+    MockColorSource(RawColor rawColor) {
+      this.rawColor = rawColor;
+    }
+
+    public RawColor getColor() {
+      return rawColor;
+    }
+  }
+
+  @Before
+  public void setup() {
+
+  }
+
+  @Test
+  public void test() {
+    ColorSubsystem cs = new ColorSubsystem(new MockColorSource(new RawColor(0, 0, 0, 0)));
+
+    ColorOptions co = cs.getColor();
+
+    assertEquals(ColorOptions.NONE, co);
+  }
+
+  @Test
+  public void testRed() {
+    RawColor rc = new RawColor(255, 153, 0, 0);
+    System.out.format("rc: Red:%d Green:%d Blue:%d\n", rc.red, rc.green, rc.blue);
```

Comment:
unit tests should not have print statements.  I used these for getting the initial test working.  They should all come out.